binder/src/main/java/io/grpc/binder/BinderChannelBuilder.java

```diff
@@ -263,14 +344,18 @@ private static final class TransportFactory implements ClientTransportFactory {
         ObjectPool<? extends Executor> offloadExecutorPool,
         SecurityPolicy securityPolicy,
         BindServiceFlags bindServiceFlags,
-        InboundParcelablePolicy inboundParcelablePolicy) {
+        InboundParcelablePolicy inboundParcelablePolicy,
+        BinderChannelCredentials channelCredentials,
```

Comment:
Make this the first argument, replacing sourcecontext?

Reply:
This should still be the second argument, just after sourceContext, and maybe move the UserHandle argument just before bindServiceFlags? (I want to keep the arguments grouped conceptually).

Same thing with the lines to set the value (the same order as the arguments).

Also, try to keep the argument order roughly consistent with the BinderChannelTransport constructor.

Reply:
Updated all the classes changed in this PR. BinderChannelCredentials will follow sourceContext and userHandle will before bindServiceFlags.